fastify.js

```diff
@@ -69,6 +69,14 @@ function build (options) {
   const requestIdLogLabel = options.requestIdLogLabel || 'reqId'
   const bodyLimit = options.bodyLimit || defaultInitOptions.bodyLimit
   const disableRequestLogging = options.disableRequestLogging || false
+  const ajvOptions = Object.assign({
+    customOptions: {},
+    useMergeAndPatch: false
+  }, options.ajv)
+
+  if (ajvOptions && typeof ajvOptions.customOptions !== 'object') {
```

Comment:
This is fragile. `ajvOptions.customOptions = []` will return true. The proper way to test if something is an object, `{}`, is `Object.prototype.toString.apply(theThing) === '[object Object]'`.

Reply:
Thanks! Fixed